Lib/fontTools/voltLib/ast.py

```diff
@@ -83,6 +114,9 @@ def __init__(self, glyph, location=None):
     def glyphSet(self):
         return (self.glyph,)
 
+    def __str__(self):
+        return f' GLYPH "{self.glyph}"'
```

Comment:
just curious, why start this and others with empty space? Is it some kind of in-built indentation?

Reply:
That is to match VOLT spacing, there are different ways to do it but this seemed to be more consistent.